### PR TITLE
Fixes firelocks being shit

### LINKED ISSUE #9359: Firelocks take in account air behind adjacent windows.
This a huge bother because when air leaks from the station any firelock standing next to a grille with glass windows will forbid people from opening it because the air inside the windows will still be at the pressure of 101 kPA, unless the windows get broken of course.
Medbay in particular suffers the most from it, you can pop out of a sleeper with exploding lungs and rising brute damage while you'll be unable to leave medbay despite the hallway behind the firelock having the same pressure as where you stand because of the air inside the glass.

I'm surprised this still wasn't a registered issue before, I think it's about time it should be adressed so they ignore the pressure in those tiles separated by directional windows.

### PR DESCRIPTION
**Pull request by 9600bauds**
Fixes #9359
Fixes THIS
mspaint_2016-05-05_00-05-47

### CODE REVIEW

File: code/game/objects/structures/false_walls.dm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 * around us, then checks the difference.
 */
 /proc/getOPressureDifferential(var/turf/loc)
-	var/minp=16777216;
+	var/minp=16777216; //What is even the significance of this number?
```
Comment (PJB3005) [inline code/game/objects/structures/false_walls.dm:14]: It's 2<sup>24</sup>, no idea how that is relevant though.

Comment (ihadtoregisterforthis) [inline code/game/objects/structures/false_walls.dm:14]: A large number
All the 24-bit RGB _True color_ variations
8<sup>8</sup>

Comment (Exxion) [inline code/game/objects/structures/false_walls.dm:14]: I was going to say it's the highest possible pressure, but then I realized I've seen higher
In other news, this function will break if the lowest-pressure adjacent tile is at a higher pressure than that number

Comment (PJB3005) [inline code/game/objects/structures/false_walls.dm:14]: Highest possible pressure should be ±3.4×10<sup>38</sup>

Comment (Exxion) [inline code/game/objects/structures/false_walls.dm:14]: According to my research, that number (2^24) is the highest value that can be stored in a 32-bit float with integer precision
So presumably pressure was SUPPOSED to be limited to that, but never actually was

Comment (clusterfack) [inline code/game/objects/structures/false_walls.dm]: Is there no way to code around this? It seems silly

Comment (Exxion) [inline code/game/objects/structures/false_walls.dm:14]: We can just
You know
Change the number in the line that all these comments are on into the actual max pressure
I'm mostly sure that wouldn't break anything

**Comment by Gringonius** [comment]
I love you so much

**Comment by 9600bauds** [comment]
Thanks, I love me too

**Pull request merged**